samples/proxy-provider-node/client/index.html

```diff
@@ -0,0 +1,98 @@
+<!-- Copyright (c) Microsoft Corporation.
+     Licensed under the MIT License. -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+  <title>Node.js Graph Proxy Client</title>
+
+  <link rel="shortcut icon" href="g-raph.png">
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <main id="main-container" role="main" class="container">
+    <h2>Login component</h2>
+    <mgt-login></mgt-login>
+    <h2>Person component</h2>
+    <mgt-person person-query="me" person-card="hover" show-presence="true" view="twolines"></mgt-person>
+    <h2>Tasks component</h2>
+    <mgt-tasks data-source="todo"></mgt-tasks>
+  </main>
+
+  <!-- MSAL -->
+  <script src="https://alcdn.msauth.net/browser/2.5.2/js/msal-browser.min.js"></script>
+
+  <!-- Graph Toolkit -->
+  <script src="https://unpkg.com/@microsoft/mgt/dist/bundle/mgt-loader.js"></script>
+
+  <!-- Configuration -->
+  <script src="config.js"></script>
+
+  <script>
+    // This is used to sign in the user and get a token that
+    // allows the client to call the proxy
+    const msalClient = new msal.PublicClientApplication(msalConfig);
```

Comment:
I'm wondering, now that I'm seeing this code, if we should package this as part of the proxy provider or a new provider? This toolkit already imports msal, so importing it twice seems unnecessary.

@jasonjoh, are you up for it?

Reply:
Hmm. I basically just imported MSAL into the client page so I could do the login to get the first token (not the Graph token). I guess what you're proposing is that we create a specialized provider that is designed to do this? Or would it make more sense to enhance the existing proxy provider to do it?

Reply:
Exactly. Let me create a new feature request for this to track it separately.

I'm leaning towards creating a new provider at the moment - reason being wanting to keep backwards compatibility of the proxy provider. But I haven't given it enough thought, so if backwards compatibility can remain while adding this functionality, keeping it all in the proxy provider makes more sense.